data/categories.txt

```diff
@@ -4763,6 +4763,318 @@ vi:Nhà vật lý trị liệu
+be:Псіхатэрапія|Псіхатэрапеўт
+bg:Психотерапия|Психотерапевт
+cs:Psychoterapie|Psychoterapeut
+da:Psykoterapi|Psykoterapeut
+de:Psychotherapie|Psychotherapeutin|Psychotherapeut
+el:Ψυχοθεραπεία|Ψυχοθεραπευτής
+es:Psicoterapia|Psicoterapeuta
+et:Psühhoteraapia|Psühhoterapeut
+eu:Psikoterapia|Psikoterapeuta
+fa:روان درمانی|روان درمانگر
+fi:Psykoterapia|Psykoterapeutti
+fr:Psychothérapie|Psychothérapeute
+he:פסיכותרפיה|פסיכותרפיסט
+hu:Pszichoterápia|Pszichoterapeuta
+id:Psikoterapi|Psikoterapis
+it:Psicoterapia|Psicoterapeuta
+ja:心理療法|サイコセラピスト
+ko:심리치료|심리 치료사
+mr:मानसोपचार|मानसोपचारतज्ज्ञ
+nb:Psykoterapi|Psykoterapeut
+nl:Psychotherapie|Psychotherapeut
+pl:Psychoterapia|Psychoterapeuta
+pt:Psicoterapia|Psicoterapeuta
+pt-BR:Psicoterapia|Psicoterapeuta
+ro:Psihoterapie|Psihoterapeut
+ru:Психотерапия|Психотерапевт
+sk:Psychoterapia|Psychoterapeut
+sv:Psykoterapi|Psykoterapeut
+sw:Tiba ya kisaikolojia|Mwanasaikolojia
+th:จิตบำบัด|นักจิตบำบัด
+tr:Psikoterapi|Psikoterapist
+uk:Психотерапія|Психотерапевт
+vi:Tâm lý trị liệu|nhà trị liệu tâm lý
+zh-Hans:心理治疗|心理治疗师
+zh-Hant:心理治療|心理治療師
+
+healthcare-sample_collection
+en:Sample collection
+ar:أخذ العينات|جمع العينات
+be:Калекцыя ўзораў
+bg:Вземане на проби|Събиране на проби
+cs:Odběr vzorků|Sběr vzorků
+da:Prøvetagning|Prøveudtagning
+de:Probenahme
+el:Δειγματοληψία|Συλλογή δείγματος
+es:Muestreo|Recogida de muestras
+et:Proovivõtmine|Proovide kogumine
+eu:Laginketa|Lagin-bilketa
+fa:نمونه برداری|مجموعه نمونه
+fi:Näytteenotto
+fr:Prélèvements biologiques|Échantillonnage
+he:דְגִימָה|אוסף דוגמאות
+hu:Mintavétel|Mintavételezés
+id:Pengambilan sampel|Pengumpulan sampel
+it:Campionamento|Raccolta di campioni
+ja:サンプリング|サンプル採取
+ko:견본 추출|샘플 수집
+mr:नमुना|नमुना संकलन
+nb:Prøvetaking|Prøvesamling
+nl:Bemonstering|Monstername
+pl:Próbkowanie|Zbieranie próbek
+pt:Amostragem|Recolha de amostras
+pt-BR:Amostragem|Coleta de amostras
+ro:Eșantionare|Colectarea probelor
+ru:Выборка|Сдача анализов
+sk:Vzorkovanie|Zber vzoriek
+sv:Provtagning|Insamling av prover
+sw:Sampuli|Mkusanyiko wa sampuli
+th:การสุ่มตัวอย่าง|เก็บตัวอย่าง
+tr:Örnekleme|Örnek toplama
+uk:Збір аналізів
+vi:lấy mẫu|Bộ sưu tập mẫu
+zh-Hans:抽样调查|样品收集
+zh-Hant:採樣|樣品採集
+
+healthcare-speech_therapist
+en:Logopedics|Speech therapist|Speech therapy
+ar:علاج النطق|معالج النطق
+be:лагапедыя|Лагапед
+bg:Логопедия|Логопед
+cs:Logopedie|Logoped
+da:Tale terapi|Talepædagog|Talerapi
+de:Logopedie|Logopädin|Logopäde
+el:Λογοθεραπεία|Λογοθεραπευτής
+es:Logopedia|Logopeda
+et:Logopeedia|Logopeed
+eu:Logopedia|Logopeda
+fa:گفتار درمانی|متخصص گفتار درمانی
+fi:Logopedia|Puheterapeutti|Logopedit
+fr:Logopédie|Orthophoniste
+he:ריפוי בדיבור|קלינאי תקשורת
+hu:Logopédia|Logopédus
+id:Logopedik|Terapis bicara
+it:Logopedia|Logopedista
+ja:ロゴペディックス|スピーチセラピスト
+ko:대화 요법|언어 치료사
+mr:स्पीच थेरपी|स्पीच थेरपिस्ट
+nb:Snakketerapi|Taleterapeut
+nl:Logopedie|Logopedist|Spraaktherapie
+pl:Logopeda|Logopedia
+pt:Logopedics|Audiologista|Logopediatria
+pt-BR:Logopedics|Fonoaudiólogo
```

Comment:
pt:Terapia da fala|Terapeuta da fala
pt-BR:Terapia da fala|Terapeuta da fala

(there is no logopedics in Portuguese anymore)

Reply:
Thanks changed in https://github.com/organicmaps/organicmaps/pull/4053/commits/dad75fe91b511fd26108fab402f7fc0a38781fe2